src/example-relay/src/Homepage/__generated__/HomepageQuery.graphql.js

```diff
@@ -10,16 +10,16 @@ import type { ConcreteRequest } from 'relay-runtime';
 type LocationsPaginatedBidirectional_data$ref = any;
 type LocationsPaginatedRefetch_data$ref = any;
 type LocationsPaginated_data$ref = any;
-export type HomepageQueryVariables = {|
```

Comment:
🤔 We should not change this file manually. Or any other generated file.

Reply:
Good point, I should have left these filtered out.

Reply:
Should be back to normal now

Reply:
I can still see some __generated__ files modified in this PR. For example: `src/example-relay/src/LocalForm/__generated__/LocalFormQuery.graphql.js`

Reply:
Sorry, I thought just running the relay-compiler was enough. They should be gone now.